### PR TITLE
[W-9143348] Enforce LWC Jest code coverage

### PR DESCRIPTION
**Pull request by spelak-salesforce**
- Enforces 100% LWC Jest code coverage among all measures except for the following block list:
    - `gauExpenditureRow`
    - `lookup`
    - `manageExpenditures`
    - `manageExpendituresTableHeader`
- We needed at least one test at 100% code coverage, so fully tested warningBanner.
    - For a11y, needed to change to only render `<h2>{message}</h2>` if there is a message.  Else, empty headers is not allowed.
    - warningBanner does not yet support translations.  Exported English labels so test can assert the proper labels are used.
- Updated `package.json` name to follow proper naming convention.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done

Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:

- [x] Any net new LWC work has JEST test coverage 50% or above
- [x] Default Sa11y tests pass for all LWC components
- [x] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary
- [x] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [ ] **All acceptance criteria have been met**
  - [x] Developer
  - [ ] Code Reviewer
- [x] Pull Request contains draft release notes
- [x] Labels, help text, and customer facing messages are reviewed by Docs
- [x] QE story level testing completed

### CODE REVIEW

**Comment by spelak-salesforce** [comment]
[W-9143348](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000AzPJYA0/view)

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "OutboundFundsNPSP",
+  "name": "outboundfundsmodulenpsp",
```
Comment (spelak-salesforce) [inline package.json:2]: `package.json` name has a required regex format.  Updated to match format.

**Comment by spelak-salesforce** [comment]
@jstvz , can you give a RelEng review?

File: force-app/main/default/lwc/warningBanner/__tests__/warningBanner.test.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+    INFO_TITLE,
+    INFO_ICON_ALTERNATIVE_TEXT,
+    CLOSE_ICON_ALTERNATIVE_TEXT,
+} from "c/warningBanner";
 
-describe("warning banner", () => {
+describe.each([
+    ["with message", `random number: ${Math.random()}`, "should display message"],
+    ["without message", null, "should not display message"],
+])("warning-banning %s", (_, message, shouldDisplayMessageTitle) => {
```
Comment (seanpat09) [inline force-app/main/default/lwc/warningBanner/__tests__/warningBanner.test.js:15]:
```suggestion
])("warning-banner %s", (_, message, shouldDisplayMessageTitle) => {
```

File: force-app/main/default/lwc/warningBanner/__tests__/warningBanner.test.js [hunk 3]
```diff
@@ -19,28 +29,187 @@ describe("warning banner", () => {
...
+            expect(closeIcons.length).toEqual(1);
+
+            const closeIcon = closeIcons[0];
+            expect(closeIcon.iconName).toEqual("utility:close");
+            expect(closeIcon.alternativeText).toEqual(CLOSE_ICON_ALTERNATIVE_TEXT);
+            expect(closeIcon.variant).toEqual("inverse");
+            expect(closeIcon.size).toEqual("x-small");
+        });
+
+        it("clicking close icon hids info alert", () => {
```
Comment (seanpat09) [inline force-app/main/default/lwc/warningBanner/__tests__/warningBanner.test.js:115]:
```suggestion
        it("clicking close icon hides info alert", () => {
```

**Review by seanpat09 — APPROVED** [review]
@spelak-salesforce just a couple of typos I found, but other than that this looks great! Didn't realize `describe.each` was a thing, that'll be a really useful!

**Review by jstvz — APPROVED** [review]

**Pull request merged**